src/metabase/models/setting.clj

```diff
@@ -481,18 +481,19 @@
         db-is-set-up? (or (requiring-resolve 'metabase.db/db-is-set-up?)
                           ;; this should never be hit. it is just overly cautious against a NPE here. But no way this
                           ;; cannot resolve
-                          (constantly false))]
+                          (constantly false))
+        db-value      #(t2/select-one-fn :value Setting :key (setting-name setting-definition-or-name))]
```

Comment:
Not really needed anymore but at first I was tweaking code here, and it's nicer to have the fetch-from-DB logic only defined once. Easier to read the rest of the code I think.